packages/editor/src/components/post-title/index.js

```diff
@@ -63,6 +69,14 @@ export default function PostTitle() {
 		};
 	}, [] );
 
+	useImperativeHandle( forwardedRef, () => ( {
+		focusTitle: () => {
+			if ( ref.current && isCleanNewPost ) {
```

Comment:
I make sure post is clean here to not requiring access to it before this function runs.